pkg/rc/replication_controller_test.go

```diff
@@ -929,3 +933,191 @@ func TestRCDoesNotFixMembership(t *testing.T) {
 		t.Fatalf("the RC should have alerted since it has some current nodes that aren't eligible and is unable to correct this. There were %d alerts", len(alerter.Alerts))
 	}
 }
+
+func TestTransferRolledBackByQuitCh(t *testing.T) {
```

Comment:
These are great tests, they do a good job of documenting the behavior we expect as well